.gitattributes

```diff
@@ -1 +1,3 @@
 * text=auto
+* text  eol=lf
```

Comment:
True that. I assume people don't blindly commit, but I changed the patch
with explicit reference to files.

On Wed, Aug 3, 2016 at 2:29 PM, Sindre Sorhus <notifications@github.com>
wrote:

> In .gitattributes
> <https://github.com/addyosmani/critical/pull/157#discussion_r73322086>:
>
> > @@ -1 +1,3 @@
> >  * text=auto
> > +* text  eol=lf
>
> But if someone adds a JPG and it becomes corrupted.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/addyosmani/critical/pull/157/files/0fa1d51b17653df34a39523462b507ea21c73a7d#r73322086>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtfVlInwEeR_CbkEJOUtaVHZvILqXks5qcHuKgaJpZM4JbMyz>
> .
>